backend/lucene/src/main/java/org/hibernate/search/backend/lucene/analysis/LuceneAnalyzerDescriptor.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.analysis;
+
+import org.hibernate.search.engine.backend.analysis.AnalyzerDescriptor;
+import org.hibernate.search.util.common.annotation.Incubating;
+
+import org.apache.lucene.analysis.Analyzer;
+
+/**
+ * The Lucene backend-specific extension of the analyzer descriptor.
+ */
+@Incubating
+public interface LuceneAnalyzerDescriptor extends AnalyzerDescriptor {
+
+	/**
+	 * @return The Lucene analyzer described by this descriptor.
+	 */
+	Analyzer analyzer();
```

Comment:
This means people will end up writing `indexDescriptor.analyzer("foo").analyzer()`, no?

Maybe `instance()` would be a better name?

Though if you're going to remove this interface, I guess it doesn't matter.